packages/assets/images/project-not-initialized.svg

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this the same as the "project-expired" svg? 🤔

Reply:
Yes, which works fine for now I think. It's the exact same hourglass we have in the async creation in the MC. So it speaks the same visuals.

Reply:
should we then rename the SVGs like we did the icons? 

~project-expeired~ hourglass?

Reply:
Do we need to have 2 identical svgs then?

Reply:
I think we're speaking of the same thing again.

1. No, we do not need two SVGs
2. Yes, using the `ProjectSuspendedSVG` in the `ProjectNotInitialized` screeen feels odd
3. Yes, we can improve the naming.
4. Yes, Duplicating the SVG seems to be the least evil also as it is code split.

However, changing the naming is a breaking change to the assets package. We can also duplicate even more to not have it breaking and export generic and specific names. Howveer, I think that's a bigger evil again even.

Reply:
Hmm ok, whatever

Reply:
Are you willing to go through a breaking change to have different namings in the assets package for the SVGs?

Reply:
Let's track is as a separate issue then, and do it in a followup as suggested by @tdeekens. We can ask design to come up with semantic names for all the assets?

Reply:
For future reference: https://github.com/commercetools/merchant-center-application-kit/issues/546